### ISSUE TITLE
Show thumbnails slider on additional monitors does not work. Error loading extension.

### ISSUE THREAD

**Issue by SKefalidis** [spin83/multi-monitors-add-on#108]
Ubuntu 19.10 clean install. It does work if I disable the extensions that come preinstalled with ubuntu, plus the 'Hide activities button' extension.

**Comment by furai** [comment]
For me the extension doesn't start up properly. @spin83 is there anything that can be done about it?
Are you still maintaining the add-on? Or should we just fork it?

**Comment by Roger-A-Rowe** [comment]
Having the same issue I think(?). This is my secondary monitor in activities overview.

![image](https://user-images.githubusercontent.com/36030802/74489933-890dea80-4e8c-11ea-90d6-f2c1374bbf6f.png)

**Comment by furai** [comment]
The error with missing hot corners should have been patched up already. You could try reinstalling the add-on if you're still experiencing issues. And the second monitor screen looks correct? Play with the add-on options.

**Comment by furai** [comment]
Actually it does not work if extension is installed from the gnome extensions site.

**Comment by Roger-A-Rowe** [comment]
Thanks for your reply.  Sorry I failed to reply. This started behaving correctly out of the blue on the desktop I was working with.  Now, I am trying to setup a laptop and am experiencing the same problem.  I have tried installing from the gnome extensions site as well as from the command line/github.  

1) The left (doesn't matter if I set as 2nd or 1st) monitor is not correct in my mind, as it does not behave like the first.  In the first monitor, the workspaces grid is fully displayed.  It is not halfway hidden like in my previous picture.

2) I've played with the add-on options.  I've removed and reinstalled multi-monitor as well as my other gnome shell extensions and restarted the computer.  Nothing seems to change this behavior. 

Anybody got ideas?
